gdsfactory/cross_section.py

```diff
@@ -175,7 +175,7 @@ class Config:
         """Configuration."""
 
         extra = "forbid"
-        fields = {
+        Annotated = {
```

Comment:
Was there something in our docs that made you want to change `fields` to `Annotated`? Fortunately I think these can all be handled by moving the `exclude=True` onto the `= Field` bits above (I'll open a PR against this branch doing that), but I'm wondering if there was something you saw that inspired this change.